pkg/operator/operator.go

```diff
@@ -376,6 +385,7 @@ func (optr *Operator) sync(key string) error {
 		{"mcs", optr.syncMachineConfigServer},
 		{"mcd", optr.syncMachineConfigDaemon},
 		{"required-pools", optr.syncRequiredMachineConfigPools},
+		{"eqg", optr.syncEtcdQuorumGuard},
```

Comment:
looking at the MCO logs sync.go logging, change eqg -> etcd-quorum-guard for clarity.

Reply:
OK